### ISSUE TITLE
Include VS templates for modules

### ISSUE THREAD

**Issue by richardszalay** [muso31/Helixbase#25]
Anders Laub's [VsTemplates](https://github.com/LaubPlusCo/LaubPlusCo.Helix.VsTemplates) VS extension supports project-local templates by placing a `.helixtemplates` in the same folder as the solution file. It can then specify a relative folder (eg. HelixTemplates) that contain the Visual Studio templates and they'll automatically appear if you create a new project using the "Helix Project" template.

What do you think about including some templates for foundation/feature/project? It would be entirely optional by the user; the folder would be ignored if they don't have/use the Helix.VsTemplates extension.

Beyond the templates + manifests, here's what we'd need in the `.helixtemplates` file:

```
templates.modules.folder=HelixTemplates
templates.modules.skipcreatedialog=true
```

**Comment by muso31** [comment]
I think it's a good idea to get some templates added, it was on the list I just never got round to it!

**Comment by marcelgruber** [comment]
This would be awesome to have some preconfigured properties on certain file types. We often see that developers misconfigure build and copy properties which can cause problems, such as deploying web.configs and breaking upstream environments.

**Comment by richardszalay** [comment]
@marcelgruber Good point, though for your specific example setting Web.config to `None` isn't necessary as as WPP only includes the Web.config from the Website project
